README.md

```diff
@@ -145,76 +148,83 @@ Optional kann je Theme mit `mainBackgroundImageURL` eine URL zum Hintergrundbild
 
 **Standard Light Farben:**
 
-```
+```json
 {
     "themes": {
         "light": {
             "mainBackgroundImageURL": "",
-            "mainBackgroundColor": "#f0f0f0",
-            "stackBackgroundColor": "#99999920",
-            "stackBackgroundColorSmall": "#99999915",
-            "stackBackgroundColorSmallTop": "#99999900",
-            "areaIconBackgroundColor": "#99999930",
-            "titleTextColor": "#222222",
-            "titleRowTextColor": "#222222",
-            "titleRowTextColor2": "#222222",
-            "smallNameTextColor": "#777777",
-            "dateTextColor": "#777777",
-            "dateTextColor2": "#777777",
-            "graphTextColor": "#888888",
-            "incidenceColorsDarkdarkred": "#941100",
-            "incidenceColorsDarkred": "#c01a00",
-            "incidenceColorsRed": "#f92206",
-            "incidenceColorsOrange": "#faa31b",
-            "incidenceColorsYellow": "#ffff64",
-            "incidenceColorsGreen": "#00cc00",
+            "mainBackgroundColor": "#f2f2f7",
+            "stackBackgroundColor": "#8e8e9320",
+            "stackBackgroundColorSmall": "#8e8e9315",
+            "stackBackgroundColorSmallTop": "#8e8e9300",
+            "areaIconBackgroundColor": "#8e8e9330",
+            "titleTextColor": "#1c1c1e",
+            "titleRowTextColor": "#1c1c1e",
+            "titleRowTextColor2": "#1c1c1e",
+            "smallNameTextColor": "#636366",
+            "dateTextColor": "#8e8e93",
+            "dateTextColor2": "#8e8e93",
+            "graphTextColor": "#636366",
+            "incidenceColorsPurple": "#af52de",
+            "incidenceColorsPink": "#ff2d55",
```

Comment:
Das Apple HIG Pink ist nicht Pink genug. Und ist vom RKI Pink etwas weit entfernt. Hier müsste eine "pinkere" alternative verwendet werden.

"RKI":
<img width="267" alt="Bildschirmfoto 2022-01-23 um 03 30 04" src="https://user-images.githubusercontent.com/97642/150682672-0112789f-a851-43ff-a028-63a73c989b53.png">
"Nachher"
<img width="277" alt="Bildschirmfoto 2022-01-23 um 03 30 40" src="https://user-images.githubusercontent.com/97642/150682674-80d57762-224d-47a5-8863-ccff9170ddbc.png">

Reply:
Wir können auch direkt die RKI Farben nehmen:

<img width="407" alt="image" src="https://user-images.githubusercontent.com/8577450/150749317-e807acf1-6740-46ff-8d7f-0abf3da36ac2.png">

Reply:
Oder als Alternativvorschlag nehmen wir:

<https://github.com/yeun/open-color>

und nehmen:

- die fünfte Spalte in der Tabelle für den Lightmode als Basis.
- die sechste Spalte in der Tabelle für den Darkmode als Basis.
- gehen je zwei bzw. vier Spalten ins tiefere Rot.

Reply:
Ich habe mal ein bisschen das Farbtool von Adobe ausgefüllt. Wie ist's hiermit?

<https://color.adobe.com/create/color-wheel>

### RKI Farben

<img width="1055" alt="image" src="https://user-images.githubusercontent.com/8577450/150798935-b6644aeb-7198-4ac5-9ac2-f4457bea1337.png">

### Aktueller Stand

> nach den Änderungen in diesem PR mit 6046f93

<img width="1062" alt="image" src="https://user-images.githubusercontent.com/8577450/150798611-7164678f-6157-43ad-8da4-5d64d81a99dc.png">

### Vorschlag

<img width="1061" alt="image" src="https://user-images.githubusercontent.com/8577450/150800195-cf02e1be-c826-4952-8312-cbbc67982adf.png">